pkg/ddl/session/session_pool.go

```diff
@@ -66,7 +66,6 @@ func (sg *Pool) Get() (sessionctx.Context, error) {
 	ctx.GetSessionVars().SetStatusFlag(mysql.ServerStatusAutocommit, true)
 	ctx.GetSessionVars().InRestrictedSQL = true
 	ctx.GetSessionVars().StmtCtx.SetTimeZone(ctx.GetSessionVars().Location())
-	infosync.StoreInternalSession(ctx)
```

Comment:
why change this part? 

cross keyspace session pool doesn't register itself to session manager, as there is no such thing now